docs/setup/install-new.md

```diff
@@ -162,6 +162,7 @@ az aks get-credentials --resource-group $resourceGroupName --name $aksClusterNam
   ```bash
   kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment.yaml
```

Comment:
we should link to AAD 1.6

Reply:
https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.6.0/deploy/infra/deployment.yaml

Reply:
I think we might have to change in other files also

Reply:
and also, we should update the installation version to 1.2.0
```
helm install ingress-azure \
      -f helm-config.yaml \
      application-gateway-kubernetes-ingress/ingress-azure
      --version 1.2.0
```

Reply:
updated